setup/pi/create-backingfiles.sh

```diff
@@ -99,9 +99,9 @@ function add_drive () {
   log_progress "Creating filesystem with label '$label'"
   if [ "$useexfat" = true  ]
   then
-    mkfs.exfat "$loopdev" -L "$label"
+    mkfs.exfat "$loopdev" -n "$label"
   else
-    mkfs.vfat "$loopdev" -F 32 -n "$label"
+    mkfs.vfat -I "$loopdev" -F 32 -n "$label"
```

Comment:
Correct, on armbian it would fail and say it needed the -I to proceed.
mkfs.vfat 4.2 (2021-01-31)
Exfat was not installed prior and i was using vfat because i knew it was in my image before starting teslausb.

Reply:
I may only needed the ignore option because i had been running the createbackingfiles.sh multiple times in testing and may have attempted to allocate storage on top of previously allocated storage without properly writing the device to image.

i would need to run again with the fixed fallocate command to see if the mkfs.vfat command works when the fallocate command works as expected

Reply:
i think this whole pull request can go away after further testing...
running into mkfs.vfat error in setup, fresh image not using armbian, using the orangepi images (4.9 kernel "orange pi 2.2 buster" debian_version 10.12) no pre-req packages or anything just following the guide, setup wifi, download install.sh and run. After first run there were some error messages about the disk currently being in use while partitioning but errors didn't halt the setup process, it rebooted promptly after first run.

Downloaded /root/bin/envsetup.sh ...
Checking that no-one is using this disk right now ... FAILED

This disk is currently in use - repartitioning is probably a bad idea.
Umount all file systems, and swapoff all swap partitions on this disk.
Use the --no-reread flag to suppress this check.

2nd boot updated the variables config with the cam/music disks then got the mkfs.vfat error.

mkfs.vfat: Partitions or virtual mappings on device '/dev/loop0', not making filesystem (use -I to override)

mkfs.fat is 4.1 (2017-01-24)
**notice it is mkfs.fat NOT mkfs.vfat**

so the fallocate command is generating a partition table and mkfs.fat doesn't like that there is a partition on the disk and refuses to write the filesystem without ignoring the warning... which, since these disks are new ignoring any warning could be a standard for the setup but also it probably needs better error handling because although the disk is empty and we are not worried about writing the partition table (because its scripted and all new files and empty and correct), maybe we can consider adding the ignore option to the make filesystem command.